packages/ibm-products/src/components/Datagrid/Datagrid/addons/Filtering/FilterPanel.js

```diff
@@ -117,7 +117,11 @@ const FilterPanel = ({
   const apply = () => {
     setAllFilters(filtersObjectArray);
     // From the user
-    onApply();
+    onApply({
```

Comment:
Since you're making some updates here with `onApply`, can you add some info about it in the docs?

Reply:
Added ✔️